configs/vite_labs.json

```diff
@@ -1,32 +1,34 @@
 {
   "index_name": "vite_labs",
   "start_urls": [
+    {
+      "url": "http://doc.vite.org/",
+      "variables": {
+        "lang": [
+          "en"
+        ]
+      }
+    },
     {
       "url": "http://doc.vite.org/(?P<lang>.*?)/",
       "variables": {
         "lang": [
-          "en",
           "zh"
         ]
       }
     }
   ],
-  "stop_urls": [
-    "/about/?$",
-    "/faq/?$",
-    "/tech/?$",
-    "/whitePaper/?$"
-  ],
+  "stop_urls": [],
```

Comment:
These values are set on purpose in order to restrain the crawl to your documentation content only. Please put it back

Reply:
I know, I have reset my site, so the `stop_url` need to change.